### ISSUE TITLE
Issue writing event data in a tribe

### ISSUE THREAD

**Issue by lythk** [eqcorrscan/EQcorrscan#381]
Hi, I'm running into an issue when creating tribes -

I'm creating a tribe for 1 day of data using an event catalog which has more events than just for that day, using tribe_construct. It cuts the templates / creates the tribe properly, but the meta data for the templates is wrong i.e. event.origin and event.picks It seems to be because it gets the event info sequentially from the catalog and not for the specific event?
This is giving me problems when I want to use the same origin location for the detections as the template. I can work around it but thought I'd report it in case its a problem on my part or it can be sorted.

cheers,
Karen

**Comment by calum-chamberlain** [comment]
Thanks for reporting this.  Can you provide a brief example so that I can try and reproduce your issue.

Can you also report the eqcorrscan version that you are running please.

CJ Chamberlain, out of office

________________________________

**Comment by calum-chamberlain** [comment]
Found it, that's an annoying bug!

Will patch shortly, thanks for reporting.

**Comment by calum-chamberlain** [comment]
Simple recreation of this bug:
```python
from obspy import UTCDateTime
from obspy.clients.fdsn import Client

from eqcorrscan import Tribe

client = Client("GEONET")

cat = client.get_events(
    latitude=-41.7, longitude=174.1, maxradius=0.5, minmagnitude=4.0,
    starttime=UTCDateTime(2013, 7, 30),
    endtime=UTCDateTime(2013, 8, 3))

st = client.get_waveforms(
    network="NZ", station="TCW", location="10", channel="EH?",
    starttime=UTCDateTime(2013, 8, 1),
    endtime=UTCDateTime(2013, 8, 2))

tribe = Tribe().construct(
    method="from_meta_file", lowcut=2, highcut=10, samp_rate=50, filt_order=4,
    length=6, prepick=0.5, st=st, meta_file=cat)

for template in tribe:
    for tr in template.st:
        matched_pick = [p for p in template.event.picks
                        if p.waveform_id.get_seed_string() == tr.id]
        if len(matched_pick) == 0:
            continue
        assert abs(matched_pick[0].time - tr.stats.starttime) == template.prepick
```

**Comment by calum-chamberlain** [comment]
Fixed in develop by #382 closing.

**Comment by calum-chamberlain** [comment]
@lythk this should be fixed in the new (0.4.1) release.  Thanks for letting me know! If you find any more issues, please raise more issues.

**Comment by lythk** [comment]
No worries. Thanks for sorting it!
